backend/db/refresh.py

```diff
@@ -57,6 +59,7 @@ def refresh_db(
         if since and dp.parse(since) > dp.parse(last_refresh) and not force_non_contiguity:
             raise ValueError(SINCE_ERR)
         since = since if since else last_refresh
+        since = dp.parse(since) - timedelta(hours=buffer_hours)
```

Comment:
### PSA regarding debugging and `refresh_status` = `active`
When I was troubleshooting this earlier, the refresh status was `active`. I don't know if that's because your local process was still running, or someone had clicked the button within the last few minutes, or maybe you exited out of your debugger without letting it complete. But yeah, if you exit debugging without letting it complete, it will still remain `active`. So we have to set it to `inactive` manually.

Maybe you already knew this. I can't remember now if I needed to set it manually to `inactive` earlier or if it resolved after a few minutes.